app/helpers/fhir_helper.rb

```diff
@@ -2,6 +2,11 @@
 
 # Helper module for FHIR translations
 module FhirHelper # rubocop:todo Metrics/ModuleLength
+  SA_EXT_BASE_URL = 'http://saraalert.org/StructureDefinition/'
```

Comment:
You can freeze these constants if they never change

Reply:
Done https://github.com/SaraAlert/SaraAlert/pull/743/commits/61893fc9539b1f03ab9f24224a8421ae9928036b

Reply:
I undid this change because rubocop told me that the `freeze` is redundant on a constant.